tests/vmi_lifecycle_test.go

```diff
@@ -1572,6 +1572,7 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 		const vmiLaunchTimeout = 360
 
 		It("soft reboot vmi with agent connected should succeed", decorators.Conformance, func() {
+			checks.SkipIfARM64(testsuite.Arch, "arm64 need ACPI to boot for UEFI boot")
```

Comment:
Not sure this is using UEFI at all btw?

Reply:
> Not sure this is using UEFI at all btw?

Yes, currently all Virtual Machine on Arm64 platform using UEFI boot.

Reply:
Okay, that makes senses. Sorry for the noise.